Source/RapyutaSimulationPlugins/Public/Robots/RRRobotStructs.h

```diff
@@ -562,12 +562,45 @@ struct RAPYUTASIMULATIONPLUGINS_API FRRRobotGeometryInfo
 };
 
 /**
- * @brief Sensor lidar info
+ * @brief Sensor info
```

Comment:
@brief Sensor info which include necessary attribute such as Topic name. Each sensor type should be childclass of this class.